src/components/Display.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Display = ({ result }) => (
+  <React.Fragment>
+    <div className="DisplayScreen">
+      <h1 className="Result">
+        {result}
+      </h1>
+    </div>
+  </React.Fragment>
+);
+
+Display.propTypes = {
+  result: PropTypes.string.isRequired,
+};
+
```

Comment:
@Salvador-ON 
[optional]
- Another way to set the default value of `result` would be to add `Display.defaultProps = {
  value: "0"
};
` to this component instead of passing it in `App` component.

Reply:
thanks i will fix that